packages/components/tooltip/src/tooltip.tsx

```diff
@@ -364,34 +379,36 @@ export const Tooltip = forwardRef<TooltipProps, "div">(
       <>
         {trigger}
 
-        <AnimatePresence>
-          {isOpen ? (
-            <Portal isDisabled={!withPortal} {...portalProps}>
-              <ui.div
-                {...getPopperProps()}
-                zIndex={resolvedZIndex}
-                pointerEvents="none"
-              >
+        <ui.div id={tooltipContentId} role="tooltip" aria-label={`${label}`}>
```

Comment:
It is added outside of the `Tooltip` , is that ok?

Reply:
I cannot determine whether it is right or not, but if we see that element `<ui.div>` as also composing the `Tooltip`, it may be right.

Reply:
I do not think this is correct. Can`t it be implemented at the `Portal` location?

Reply:
You mean like this?
```tsx
 <Portal id={tooltipContentId} role="tooltip" aria-label={`${label}`} isDisabled={!withPortal} {...portalProps}>
    // content
 </Portal>
```

Reply:
Yes, if possible.

Reply:
I understand your point, but even if I implement it like that, the problem will not be resolved. It means the `<Portal />` does not exist from the beginning because of the conditional rendering, and these `id={tooltipContentId} role="tooltip" aria-label={`${label}`}` attributes does not exist either. So, in this PR, I take this approach.

![image](https://github.com/yamada-ui/yamada-ui/assets/107479598/478dafc4-51a7-40f5-9dce-b3a185ce4173)

Reply:
Discuss what to do with the accessibility team.

Reply:
Ok, Thanks!